erpnext_germany/custom_fields.py

```diff
@@ -162,6 +162,35 @@ def get_custom_fields():
 				"translatable": 0,
 			},
 		],
+		"Purchase Invoice": [
+			{
+				"fieldtype": "Link",
+				"fieldname": "business_trip",
+				"label": _("Business Trip"),
+				"options": "Business Trip",
+				"insert_after": "apply_tds",
+			},
+			{
+				"fieldtype": "Link",
+				"fieldname": "business_trip_employee",
+				"label": _("Business Trip Employee"),
+				"options": "Employee",
+				"insert_after": "business_trip",
+				"read_only": 1,
+				"fetch_from": "business_trip.employee",
+				"depends_on": "eval: doc.business_trip",
+				"ignore_user_permissions": 1,
+			},
```

Comment:
```suggestion
			{
				"fieldtype": "Link",
				"fieldname": "business_trip_employee",
				"label": _("Business Trip Employee"),
				"options": "Employee",
				"insert_after": "business_trip",
				"read_only": 1,
				"fetch_from": "business_trip.employee",
				"depends_on": "pay_to_employee",
				"ignore_user_permissions": 1,
			},
```

Employee seems only relevant if they are supposed to get paid. Update field order?

Reply:
I think, the field also has a bit of a descriptive function (even if the checkbox is unchecked)